phys/module_sf_noahmplsm.F

```diff
@@ -8027,8 +8027,8 @@ SUBROUTINE SPRINKLER_IRRIGATION(parameters,NSOIL,DT,SH2O,SMC,SICE,& !in
     ELSE ! Equation (4)
        IRRLOSS   = 4.337*(EXP(0.077*WINDSPEED))*(((ESAT1-EAIR)*0.01)**(-0.098)) ! [%]
     END IF
-    IF(ISNAN(IRRLOSS))IRRLOSS=4.0 ! In case if IRRLOSS is NaN
-    
+    !IF(ISNAN(IRRLOSS))IRRLOSS=4.0 ! In case if IRRLOSS is NaN
```

Comment:
@prasanthvkrishna 
It is possible that you should keep the existing "if(isnan(irrlos))" test. The standard does not guarantee what happens if you try to do a arithmetic test on a NaN value. I think BOTH tests should be kept.

Reply:
Hi Dave, 

We are in the process to make Noah MP consistent across different models. @Cenlin_He has reported that he is getting HRLDAS compile errors while using the PGI compiler. This was the reason I removed the ISNAN check. Do the WRF compile test considers PGI? Please let me know.
Thanks

Reply:
Hi @davegill 
ISNAN is not supported by PGI compiler (at least during my test in cheyenne). In order to avoid limiting the use of Noah-MP source code across different host models and compiler platforms, I think it is better to remove the ISNAN check or use something equivalent but works for different compilers. 
Also, based on the current code, it seems that the IRRLOSS variable is unlikely to have NaN values. 
What do you think?

Reply:
I am OK with this decision